### PR TITLE
DOC: Docs & tutorial updates for ingesting, beginners, and csvdir

### PR DESCRIPTION
**Pull request by freddiev4**
Added some much needed doc updates here and also a mini tutorial for using `.csv` files for your bundles (x-ref https://github.com/quantopian/zipline/issues/2060)

### CODE REVIEW

**Comment by ernestoeperez88** [comment]
One observation unrelated to your changes, I would remove/reword the live-trading support sentence at the beginning of the README.rst file.

File: README.rst [hunk 1]
```diff
@@ -18,22 +18,22 @@ engine powering `Quantopian <https://www.quantopian.com>`_ -- a free,
...
+- **"Batteries included":** many common statistics like
   moving average and linear regression can be readily accessed from
   within a user-written algorithm.
-- Input of historical data and output of performance statistics are
+- **PyData integration:** Input of historical data and output of performance statistics are
   based on Pandas DataFrames to integrate nicely into the existing
-  PyData eco-system.
-- Statistic and machine learning libraries like matplotlib, scipy,
+  PyData ecosystem.
+- **Statistics and Machine Learning libraries:** matplotlib, scipy,
```
Comment (ernestoeperez88) [inline README.rst]: I would reword this to explain how Zipline uses or interacts with these libraries.

File: README.rst [hunk 2]
```diff
@@ -139,15 +139,15 @@ The following code implements a simple dual moving average algorithm.
                long_mavg=long_mavg)
 
 
-You can then run this algorithm using the Zipline CLI. From the command
-line, run:
+You can then run this algorithm using the Zipline CLI; you'll need a `Quandl <https://www.quandl.com/>`__ API key for default data bundle.
```
Comment (ernestoeperez88) [inline README.rst]: Might want to link instead to the "Getting an API key" section of Quandl's docs:

https://docs.quandl.com/docs#section-authentication

Comment (ernestoeperez88) [reply README.rst]: "API key for default data bundle." -> "API key to download the default data bundle."

File: README.rst [hunk 3]
```diff
@@ -139,15 +139,15 @@ The following code implements a simple dual moving average algorithm.
...
+Once you have you key, rom the command line, run:
```
Comment (ernestoeperez88) [inline README.rst]: "Once you have you key, rom the command line, run:" -> "Once you have your key, run the following from the command line:"

File: docs/source/beginner-tutorial.rst [hunk 4]
```diff
@@ -126,38 +127,41 @@ on OSX):
...
-                                     is '-' the perf will be written to stdout.
-                                     [default: -]
-     --print-algo / --no-print-algo  Print the algorithm to stdout.
-     --help                          Show this message and exit.
+	 Usage: zipline run [OPTIONS]
+
+  Run a backtest for the given algorithm.
+
+	Options:
+   -f, --algofile FILENAME         The file that contains the algorithm to run.
```
Comment (richafrank) [inline docs/source/beginner-tutorial.rst]: Looks like the indentation here got messed up.

File: README.rst [hunk 5]
```diff
@@ -139,15 +139,15 @@ The following code implements a simple dual moving average algorithm.
...
 
 .. code:: bash
 
-    $ zipline ingest
-    $ zipline run -f dual_moving_average.py --start 2011-1-1 --end 2012-1-1 -o dma.pickle
+    $ QUANDL_API_KEY=<yourkey> zipline ingest -b quandl
+    $ zipline run -f dual_moving_average.py --start 2014-1-1 --end 2018-1-1 -o dma.pickle
 
-This will download the AAPL price data from `quantopian-quandl` in the
+This will download the AAPL price data from `quandl` in the
```
Comment (ernestoeperez88) [inline README.rst]: I would generalize this a bit: 

"This will download asset pricing data from `quandl`, and stream it through the algorithm over the specified time range. Then, the resulting performance DataFrame is saved in dma.pickle, which you can load and analyze from within Python.

File: docs/source/beginner-tutorial.rst [hunk 6]
```diff
@@ -101,14 +101,15 @@ interfaces: A command-line interface, ``IPython Notebook`` magic, and
 
 Ingesting Data
 ^^^^^^^^^^^^^^
-If you haven't ingested the data, run:
+If you haven't ingested the data, you'll need a `Quandl <https://www.quandl.com/>`__ API key to
```
Comment (ernestoeperez88) [inline docs/source/beginner-tutorial.rst]: Same comment as before, we might want to link to https://docs.quandl.com/docs#section-authentication instead.

File: docs/source/bundles.rst [hunk 7]
```diff
@@ -47,15 +47,15 @@ process will invoke some custom bundle command and then write the data to a
 standard location that zipline can find. By default the location where ingested
 data will be written is ``$ZIPLINE_ROOT/data/<bundle>`` where by default
 ``ZIPLINE_ROOT=~/.zipline``. The ingestion step may take some time as it could
-involve downloading and processing a lot of data. This can be run with:
+involve downloading and processing a lot of data. You'll need a
+`Quandl <https://www.quandl.com/>`__ API key to ingest the default bundle. This can be run with:
```
Comment (ernestoeperez88) [inline docs/source/bundles.rst]: We might want to link to "Getting an API key" section of Quandl docs, https://docs.quandl.com/docs#section-authentication

**Comment by freddiev4** [comment]
@ernestoeperez88 should be good for another review

File: README.rst [hunk 8]
```diff
@@ -13,28 +13,28 @@
 |Coverage Status|
 
 Zipline is a Pythonic algorithmic trading library. It is an event-driven
-system that supports both backtesting and live-trading. Zipline is currently used in production as the backtesting and live-trading
+system that for backtesting. Zipline is currently used in production as the backtesting and live-trading
```
Comment (ernestoeperez88) [inline README.rst]: "It is an event-drive system that for" -> "It is an event driven system for backtesting"

File: README.rst [hunk 9]
```diff
@@ -139,18 +139,17 @@ The following code implements a simple dual moving average algorithm.
                long_mavg=long_mavg)
 
 
-You can then run this algorithm using the Zipline CLI. From the command
-line, run:
+You can then run this algorithm using the Zipline CLI; you'll need a `Quandl <https://docs.quandl.com/docs#section-authentication>`__ API key to ingest the default data bundle.
+Once you have you key, run the following from the command line:
```
Comment (ernestoeperez88) [inline README.rst]: "Once you have you key" -> "Once you have your key"

**Comment by ernestoeperez88** [comment]
@freddiev4 left comments on a couple of typos, otherwise LGTM.

**Comment by freddiev4** [comment]
@ernestoeperez88 ok sweet. Will update and then merge 🙂
